src/pombot/commands/stop.js

```diff
@@ -2,25 +2,25 @@
  * The `stop` command, which stops a Pom.
  */
 import {createCommand} from 'chatter';
-import states from '../states';
+import {states} from '../pomConfig';
 
 export default function(pom) {
   return createCommand({
     name: 'stop',
     description: 'Stops the current pom.',
-  }, message => {
+  }, (message, {channel}) => {
     switch (pom.state) {
       case states.RUNNING:
       case states.ON_BREAK:
         // if pom is running, get time left stop it
-        const timeLeft = pom.getMinutes(pom.timeLeft);
+        const timeLeft = pom.getTimeString(pom.timeLeft);
         pom.stop();
-        return `pom stopped; there were *${timeLeft} minutes* remaining.`;
+        return `🍅 pom stopped – there were *${timeLeft}* remaining.`;
         break;
       default:
         // if pom is not running
-        // TODO get @sender & set conditional for which start command to use (dm vs room)
-        return 'there is no pom currently running; I can start one if you ask me to with `pom start`';
+        const command = (channel.is_im) ? '`start`' : '`pom start`';
```

Comment:
I'm slightly concerned about programmatically determining the command name like this. What if the top-level bot name was changed to `pombot`? Then code like this will need to be updated anywhere it exists.

If you returned `false` here, the usage information for the `stop` command would be shown, which would look something like:

> Usage: `pom stop`
> Or try **help pom** for more information.

Is it important to be be able to know the name of different commands?